app/src/main/java/io/homeassistant/companion/android/sensors/SettingsSensorManager.kt

```diff
@@ -0,0 +1,73 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.provider.Settings
+import android.util.Log
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.common.R as commonR
+
+class SettingsSensorManager : SensorManager {
+    companion object {
+        private const val TAG = "ScreenBrightness"
+
+        val screenBrightness = SensorManager.BasicSensor(
+            "screen_brightness",
+            "sensor",
+            commonR.string.basic_sensor_name_screen_brightness,
+            commonR.string.sensor_description_screen_brightness,
+        )
+    }
+
+    override fun docsLink(): String {
+        return "https://companion.home-assistant.io/docs/core/sensors#screen-brightness-sensor"
+    }
+
+    override val enabledByDefault: Boolean
+        get() = false
+    override val name: Int
+        get() = commonR.string.sensor_name_device_settings

```

Comment:
While I understand this name from a technical poin of view, it doesn't make any sense to me as a normal user. Aren't most of the sensors showing the state of a setting and as a result "Device Settings Sensors"?

Because there isn't a lot of logic to 'support' the `Settings.System` API, isn't it better to call this one something like "Display Sensors" instead?

Reply:
My thoughts was that we may be showing more values from `Settings.*` in here, I can change it if we feel the only things of value are screen related settings? In reality we are just reading the raw values of the pre-defined constants, whatever they may be.

Reply:
But will users understand that it is from `Settings.*`? That shouldn't be required in my opinion and as a result the name isn't good. 

(We can get vibration or WiFi or font scaling info from the same API but if the app were to all add it here it wouldn't be a logical group for non-tech people.)

Reply:
Thats a good point, will update to "Display Sensors" since thats more correct here